Makefile

```diff
@@ -7,6 +7,8 @@ IMAGE_TAG_BASE ?= project-flotta.io/flotta-operator
 
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
+HTTP_IMG ?= http-api:latest
```

Comment:
WDYT of referring to it as `flotta-api-server` ?

Reply:
I was trying to avoid that name not to make an impression that it is a k8s API server.